Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1483,7 +1509,9 @@ - (void)serverCallback:(BNCServerResponse *)response {
             } else {
                 [BNCPreferenceHelper setSessionParams:NO_STRING_VALUE];
             }
-            
+            if ([BNCPreferenceHelper getNeedAppListCheck]) {
```

Comment:
I was misreading the upload on line 1171 and the setAppListCheckDone at 1618 incorrectly, my mistake.